src/fights.ts

```diff
@@ -323,157 +319,137 @@ function bestWitchessPiece() {
-            1 + get("_pocketProfessorLectures") - startLectures;
+              macro: macro,
+            });
+            embezzlerLog.initialEmbezzlersFought +=
+              1 + get("_pocketProfessorLectures") - startLectures;
+          }
+          set(property, true);
+          postCombatActions();
+          startWandererCounter();
         }
-        set("_garbo_weightChain", true);
-        postCombatActions();
       }
 
-      startWandererCounter();
-
       // REMAINING EMBEZZLER FIGHTS
       let nextFight = getNextEmbezzlerFight();
       while (nextFight !== null) {
         const startTurns = totalTurnsPlayed();
         if (have($skill`Musk of the Moose`) && !have($effect`Musk of the Moose`)) {
           useSkill($skill`Musk of the Moose`);
         }
-        if (["Macrometeorite", "Powerful Glove"].includes(nextFight.name)) {
+        if (nextFight.gregariousReplace) {
           saberCrateIfDesired();
         }
         withMacro(embezzlerMacro(), () => {
           if (nextFight) {
-            useFamiliar(meatFamiliar());
-            if (
-              (have($familiar`Reanimated Reanimator`) || have($familiar`Obtuse Angel`)) &&
-              get("_badlyRomanticArrows") === 0 &&
-              !nextFight.draggable
-            ) {
-              if (have($familiar`Obtuse Angel`)) useFamiliar($familiar`Obtuse Angel`);
-              else useFamiliar($familiar`Reanimated Reanimator`);
-            }
-
-            if (
-              nextFight.draggable &&
-              !get("_envyfishEggUsed") &&
-              (booleanModifier("Adventure Underwater") || waterBreathingEquipment.some(have)) &&
-              (booleanModifier("Underwater Familiar") ||
-                familiarWaterBreathingEquipment.some(have)) &&
-              (have($effect`Fishy`) || (have($item`fishy pipe`) && !get("_fishyPipeUsed"))) &&
-              !have($item`envyfish egg`) &&
-              mallPrice($item`pulled green taffy`) < 10000 &&
-              retrieveItem($item`pulled green taffy`)
-            ) {
-              setLocation($location`The Briny Deeps`);
-              meatOutfit(true, Requirement.merge(nextFight.requirements), true);
-              if (get("questS01OldGuy") === "unstarted") {
-                visitUrl("place.php?whichplace=sea_oldman&action=oldman_oldman");
-              }
-              if (!have($effect`Fishy`)) use($item`fishy pipe`);
-              nextFight.run({ location: $location`The Briny Deeps` });
-            } else if (nextFight.draggable) {
-              const type =
-                nextFight.name === "Backup" ? draggableFight.BACKUP : draggableFight.WANDERER;
-              const location = determineDraggableZoneAndEnsureAccess(type);
-              setLocation(location);
-              meatOutfit(true, Requirement.merge(nextFight.requirements));
-              nextFight.run({ location });
+            const romanticFamiliar = $familiars`Obtuse Angel, Reanimated Reanimator`.find(have);
```

Comment:
any time i see `.find(have)` it sets off  kill bill sirens, but this is fine for familiars